.github/workflows/linters.yml

```diff
@@ -0,0 +1,20 @@
+ 
+name: Linters
+
+on: pull_request
+
+jobs:
+  rubocop:
+    name: Rubocop
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-ruby@v1
+        with:
+          ruby-version: 2.6.x
+      - name: Setup Rubocop
+        run: |
+          gem install --no-document rubocop:'~>0.81.0' # https://docs.rubocop.org/en/stable/installation/
+          [ -f .rubocop.yml ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/ruby/.rubocop.yml
```

Comment:
You are not using the same `.rubocop.yml` file that Microverse requires. Please add the correct one to the root directory.